composer.json

```diff
@@ -16,11 +16,17 @@
     "require-dev": {
         "phpunit/phpunit": ">=4.8",
         "squizlabs/php_codesniffer": "^2.5",
-        "jakub-onderka/php-parallel-lint": "^0.9.2"
+        "jakub-onderka/php-parallel-lint": "^0.9.2",
+        "phing/phing": "^2.13"
     },
     "autoload": {
         "psr-4": {
             "Scientist\\": "src"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "": "tests"
+        }
```

Comment:
hi!
about possible fractals modelization 
observed in some works composed by WA Mozart